src/cli.cr

```diff
@@ -2,6 +2,8 @@ require "option_parser"
 require "./commands/*"
 
 module Shards
+  class_property? display_help : Bool = false
```

Comment:
**question:** What's the purpose of using a class variable?
This property is entirely scoped to the `.run` method, so it looks like it could be a local variable there?